pkg/server/server.go

```diff
@@ -47,6 +47,7 @@ const (
 	meteringProxyEndpoint          = "/api/metering"
 	customLogoEndpoint             = "/custom-logo"
 	helmChartRepoProxyEndpoint     = "/api/helm/charts/"
+	gitOpsEndpoint                 = "/api/gitops"
```

Comment:
nit: The convention is to use `gitops` when it's the first part of a camel-case variable name (like we use `openshift` instead of `openShift` and `oauth` instead of `oAuth`).

```suggestion
	gitopsEndpoint                 = "/api/gitops"
```